### DISCUSSION TITLE
无法选择除中英文模型之外的模型进行打标

### DISCUSSION THREAD

**Discussion by afs1955** [PFCCLab/PPOCRLabel#25] [Q&A]
![image](https://github.com/PFCCLab/PPOCRLabel/assets/98142355/d85a3532-5b4f-46c1-bb70-352b84d267f0)
如图，在选择模型之后，确实会下载对应的模型，但是马上就会提示 ppocr ERROR: lang korean is not support, we only support dict_keys(['en', 'ch']) for layout models，如下图：
![image](https://github.com/PFCCLab/PPOCRLabel/assets/98142355/3d1c2a1f-3eb2-491f-be3b-00ee3d316f50)

**Comment by TTTalen** [comment]
仅提供一个可行的方案。如果本地保存模型的话，可以尝试在PPOCRLabel.py内的self.ocr = PaddleOCR中增加本地模型的本地路径。

**Comment by GreatV** [comment]
@afs1955 如果方便的话，你可以提交一个PR来修复这个问题。
